src/cli/self_update/unix.rs

```diff
@@ -92,6 +92,13 @@ pub(crate) fn do_add_to_path() -> Result<()> {
                 _ => &source_cmd,
             };
 
+            let rc_dir = rc.parent().with_context(|| {
+                format!(
+                    "could not find parent directory for rcfile path: `{}`",
```

Comment:
Nit: maybe "could not find parent directory" -> "no parent directory"? This isn't actually about "finding" the directory (in the filesystem), but about an rcfile path having no directories at all.

Reply:
Done!